server/app.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const {} = require("./controllers/controllers.errors");
```

Comment:
What's this line doing?

Reply:
We should only import/require things as and when we need them, I'd recommend to remove this until the time comes that you wish/want to require functions from the errors file